README.md

```diff
@@ -39,7 +46,7 @@ Run the following command to wait until the fybrikapplication be ready.
 ```bash
 while [[ $(kubectl get fybrikapplication my-notebook -n default -o 'jsonpath={.status.ready}') != "true" ]]; do echo "waiting for FybrikApplication" && sleep 5; done
 ```
-There should be four running pods in `fybrik-blueprints` namespace.
+
```

Comment:
we can add some kubectl wait command to "Wait For the pod `my-notebook-default-dremio-module-xxxx` to be completed. "